scripts/wiki.coffee

```
@@ -0,0 +1,40 @@
+Github = require 'github'
+
+module.exports = (robot) ->
+
+  robot.respond /(?:wiki) (.*)/i, (message) ->
+    GITHUB_TOKEN = ''
+    if process.env.GITHUB_TOKEN
+      GITHUB_TOKEN = process.env.GITHUB_TOKEN
+    else
+      return robot.emit 'error: No GITHUB_TOKEN defined in environment'
```

I think we should throw the error if GITHUB_TOKEN is not provided :-)